server/db.js

```diff
@@ -615,7 +615,7 @@ var resetAllSettings = function (callback) {
         defaultSettings.question['defaultMinPoints'] = 10;
         defaultSettings.question['defaultMaxPoints'] = 100;
         defaultSettings.question['timeoutEnabled'] = true;
-        defaultSettings.question['timeoutPeriod'] = 1;
+        defaultSettings.question['timeoutPeriod'] = 30*60000;
```

Comment:
So the default now is 30 minutes?

Reply:
yes sir, I just put that for now, we can change it later